lib/src/mixins/batched_redraws.dart

```diff
@@ -39,6 +41,9 @@ class _RedrawScheduler implements Function {
 
         (component as react.Component)?.setState({}, chainedCallbacks);
       })
+      ..forEach((component, callbacks) {
```

Comment:
Sure thing. We want to unset this flag only after all of the batched redraws (and the redraws they cause by rerendering their descendants) have completed. 

Otherwise, if we just unset the flag after a component redraw, it's possible that subsequent redraws in the batch will cause redrawing of that component again.